src/chainbin/nnet3-chain-get-egs.cc

```diff
@@ -190,8 +206,10 @@ static bool ProcessFile(const fst::StdVectorFst &normalization_fst,
         ivector_frame = ivector_feats->NumRows() - 1;
       Matrix<BaseFloat> ivector(1, ivector_feats->NumCols());
       ivector.Row(0).CopyFromVec(ivector_feats->Row(ivector_frame));
+      KALDI_ASSERT(ivector_feats->NumCols() % num_cmn_offsets == 0);
       NnetIo ivector_io("ivector", 0, ivector);
-      nnet_chain_eg.inputs[1].Swap(&ivector_io);
+      int32 ivec_ind = (cmn_offsets ? 2 : 1);
```

Comment:
please don't use 'ind' as an abbreviation for index; the Google style guide frowns on abbreviations, and this particular abbreviation is never seen in the code.  you can call it ivector_index.  please fix your other code if this issue exists there too.